src/main/java/org/pentaho/metastore/stores/xml/XmlMetaStore.java

```diff
@@ -226,7 +226,9 @@ protected void lockStore() throws MetaStoreException {
       }
       totalTime += 100;
       if ( totalTime > 10000 ) {
-        throw new MetaStoreException( "Maximum wait time of 10 seconds exceed while acquiring lock" );
+        throw new MetaStoreException( "Maximum wait time of 10 seconds exceed while acquiring lock. "
+          + "If there is only one instance of this application running, "
+          + "try deleting the '.lock' file in the metastore folder if the problem re-occurring on startup" );
```

Comment:
"if the problem re-occuring on startup" needs to be adjusted. 
maybe "reoccurs" (no hyphen) instead of "re-occuring".

Reply:
addressed via https://github.com/pentaho/metastore/commit/da8c0e732a9d9b5c88b98fbcf558c0c20a6e0d67